Directory.Packages.props

```diff
@@ -16,7 +16,7 @@
     <PackageVersion Include="CoreForms.Web.Infrastructure.Roslyn" Version="$(CoreFormsVersion)" />
     <PackageVersion Include="CoreForms.Web.Services" Version="$(CoreFormsVersion)" />
     <PackageVersion Include="Fork.Coypu" Version="3.4.0" />
-    <PackageVersion Include="JetBrains.Annotations" Version="2023.3.0" />
+    <PackageVersion Include="JetBrains.Annotations" Version="2024.3.0" />
```

Comment:
Since there are changes, please also update the JetBrains-file in the SharedSources folder. It should be straight forward and I dont need to re-see it.